CompilerSource/parsing/pretty_printer.cpp

```diff
@@ -324,6 +340,9 @@ bool AST::CppPrettyPrinter::VisitReturnStatement(AST::ReturnStatement &node) {
   if (node.expression) {
     VISIT_AND_CHECK(node.expression);
   }
+  if (node.is_exit) {
+    print("0");
+  }
```

Comment:
Our scripts return `variant`, so if `exit` is translated to `return;`, it will generate an error. Instead, it should be `return 0;`, as it is in the master branch.